test/lib/code_corps/github/sync/sync_test.exs

```diff
@@ -139,6 +153,24 @@ defmodule CodeCorps.GitHub.SyncTest do
       assert Repo.aggregate(User, :count, :id) == 13
     end
 
+    # coupled to fixtures. depends on
+    # - fixtures/github/endpoints/issues.json on having at least 4 issues
+    #   linked to pull requests
+    # - fixtures/github/endpoints/pulls.json having payloads for those 4 pull
+    #   requests (matched by "number")
+    test "matches github issue with github pull request correctly" do
+      {:ok, github_repo} = setup_test_repo() |> Sync.sync_repo
+
+      %GithubRepo{github_issues: github_issues} =
+        GithubRepo |> Repo.get(github_repo.id) |> Repo.preload(:github_issues)
+
+      linked_issues =
+        github_issues
+        |> Enum.filter(fn i -> not is_nil(i.github_pull_request_id) end)
```

Comment:
Prefer `Enum.reject` here, so you can have a positive predicate. Then you can call it `linked_issue_count` and move the `Enum.count` up outside the `assert`.